samples/nrf9160/http_update/full_modem_update/boards/nrf9160dk_nrf9160_ns.overlay

```diff
@@ -9,6 +9,7 @@
 	mx25r64: mx25r6435f@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
+		status = "okay";
```

Comment:
is this needed here?
the external flash was disabled for `nrf9160dk_nrf9160_common_0_14_0` versions.
this is the default board, with no version specified, so the 0.14.0 change won't get included in the default board build.

same comment for other default board overlays.

Reply:
It is not needed when building for the default board. However, if the board revision is specified, and >=0.14.0, it will be disabled.

Reply:
Thanks for pointing this out, I had not tried this sample with a version. It should be fixed now!